### ISSUE TITLE
User Permissions: Delete

### ISSUE THREAD

**Issue by ta2edchimp** [ta2edchimp/BattleReporter#35]
Fetch a character's role in the `BR_OWNERCORP` to check if permission to delete BattleReports may be granted.

**Comment by ta2edchimp** [comment]
It would require configuring a Corporation API-Key providing access to `MemberSecurity`.

A Call to `https://api.eveonline.com/corp/MemberSecurity.xml.aspx?keyID=xxx&vCode=yyy` would return something like this:

```xml
    <result>
        <row characterID="123456789" name="Some Dude">
            <rowset name="roles" key="roleID" columns="roleID,roleName">
                <row roleID="1" roleName="roleDirector"/>
                ...
            </rowset>
            ...
        </row>
    </result>
```

A character's role required to delete BattleReports would be sth. like _Director_ / `roleDirector`.

**Comment by ta2edchimp** [comment]
Closed by 433a27c
